server/test/integration/serverTest.js

```diff
@@ -16,8 +16,7 @@ const imports = require('../../server.js')
 const server = imports.server
 const db = imports.db
 
-// eslint-disable-next-line func-style
-const sleep = millis => new Promise(resolve => setTimeout(resolve, millis))
```

Comment:
Removed this to use `helpers.sleep` instead. It has the exact same implementation.